mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
> #3672 is supposed to take care of this by using robust scaler instead of the z-score.

Indeed, this is what I thought. I didn't realize the PR wasn't merge and it came up in a diff in this PR!